doc/source/development/testing.rst

```diff
@@ -93,6 +93,28 @@ Another tool available on github is `cstar_perf <https://github.com/datastax/cst
 
 CircleCI
 --------
-Cassandra ships with a default `CircleCI <https://circleci.com>`_ configuration, to enable running tests on your branches, you need to go the CircleCI website, click "Login" and log in with your github account. Then you need to give CircleCI permission to watch your repositories. Once you have done that, you can optionally configure CircleCI to run tests in parallel - click "Projects", then your github account and then click the settings for the project. If you leave the parallelism at 1 for Cassandra, only ``ant eclipse-warnings`` and ``ant test`` will be run. If you up the parallelism to 4, it also runs ``ant long-test``, ``ant test-compression`` and ``ant stress-test``
+Cassandra ships with a default `CircleCI <https://circleci.com>`_ configuration, to enable running tests on your branches, you need to go the CircleCI website, click "Login" and log in with your github account. Then you need to give CircleCI permission to watch your repositories. Once you have done that, you can optionally configure CircleCI to run tests in parallel - click "Projects", then your github account and then click the settings for the project. If you leave the parallelism at 1 for Cassandra, only ``ant eclipse-warnings`` and ``ant test`` will be run. If you up the parallelism to 4, it also runs ``ant long-test``, ``ant test-compression`` and ``ant stress-test``.
+
+The configuration for CircleCI is in the ``.circleci/config.yml`` file. This configuration file is meant to use low resources, you can find equivalent configuration files using more resources in the same ``.circleci`` directory. Please read the ``readme.md`` file in that directory for further information.
+
+The optional ``repeated_utest``/``repeated_dtest`` CircleCI jobs run a specific JUnit/Python test repeatedly. This is useful to verify that a certain test is stable. It's usually a good idea to run these jobs when adding or modifying a test. To specify what test should be run and the number of repetitions you should edit the related evironment variables in the CircleCI configuration file:
+
++----------------------------+-------------------------------------------------------------------------+
+| Variable                   | Description                                                             |
++============================+=========================================================================+
+| ``REPEATED_UTEST_TARGET``  | The Ant test target to run                                              |
++----------------------------+-------------------------------------------------------------------------+
+| ``REPEATED_UTEST_CLASS``   | The name of JUnit class to be run multiple times                        |
++----------------------------+-------------------------------------------------------------------------+
```

Comment:
Instead of JUnit I would probably still use UTest class or so

Reply:
I have used "Java" instead. I have also added some examples and changed the header level of the CircleCI section.

Reply:
Thank you for doing it! Looks great!